tauv_common/src/vision/config/robosub.names

```diff
@@ -0,0 +1,4 @@
+dice1
```

Comment:
ideally these config things should be moved into the tauv_config folder, but I think this is ok for now.